### PR TITLE
Adding hraVersion to Excel download

### PR DESCRIPTION
**Pull request by Knight7561**

### CODE REVIEW

File: projects/v2/src/app/components/report/report.component.ts [hunk 1]
```diff
@@ -139,17 +140,18 @@ export class ReportComponent implements OnInit, AfterViewInit {
...
     const biomarkerCols = [];
     biomarkersSeperateNames.forEach((bm) => {
-      if (this.displayedColumns.includes(bm.name) === false){
+      if (this.displayedColumns.includes(bm.name) === false) {
         biomarkerCols.push(bm.name);
-      }    
+      }
       this.displayedColumns = [
         'organName',
+        'tableVersion',
```
Comment (bherr2) [inline projects/v2/src/app/components/report/report.component.ts:154]: Table version does not need to be shown in the UI, only in the xlsx export.

Comment (bherr2) [reply projects/v2/src/app/components/report/report.component.ts:154]: Column needs to be `table_version` and not `tableVersion`

File: projects/v2/src/app/modules/root/root.component.ts [hunk 2]
```diff
@@ -204,6 +206,7 @@ export class RootComponent implements OnInit, OnDestroy {
       const comparisonHasFile = query.get('comparisonHasFile');
       const sheet = query.get('sheet');
       const playground = query.get('playground');
+      const hraVersionFromQuery = query.get('hraVersions');
```
Comment (bherr2) [inline projects/v2/src/app/modules/root/root.component.ts:209]: Why hraVersions? Let's talk before/after standup.

**Review by bherr2 — CHANGES_REQUESTED** [review]

File: projects/v2/src/app/modules/root/root.component.ts [hunk 3]
```diff
@@ -633,4 +636,9 @@ export class RootComponent implements OnInit, OnDestroy {
         });
     }
   }
+
+  extractNameFromFull(name: string) {
```
Comment (pbrm4) [inline projects/v2/src/app/modules/root/root.component.ts:640]: extractNameFromFull -> extractOrganNameFromFull

File: projects/v2/src/app/components/report/report.component.ts (hunk 1 above)
Comment (bherr2) [reply projects/v2/src/app/components/report/report.component.ts:154]: I don't see any table_version string in your code. Can you explain?

Comment (bherr2) [reply projects/v2/src/app/components/report/report.component.ts:154]: Also, I made a big oops. It IS actually supposed to be `tableVersion` in the sheets. Not sure if that was a change on the humnanatlas.io site or I just totally flubbed it. My bad. Please change back to `tableVersion`.

File: projects/v2/src/app/modules/root/root.component.html
Comment (pbrm4) [inline projects/v2/src/app/modules/root/root.component.html:1]: Does this file have any changes? Looks like it has only formatting changes

Comment (Knight7561) [reply projects/v2/src/app/modules/root/root.component.html:1]: YEAH

**Pull request merged**